### ISSUE TITLE
Agensgraph over PostgreSQL (extension)

### ISSUE THREAD

**Issue by AnwarsaeedDMU** [bitnine-oss/agensgraph#620]
I am trying to run agensgraph on the relational data in my existing PostgreSQL (localhost), I tried to convert all data with FDW extension before its work but now I want use the hybrid querying to keep some data in relational database and some in Agesgraph. I tried different techniques but its not work for me. I installed agensgraph 2.12 and 2.13 but I didn't found how to connect extension or I have to install it inside PostgreSQL directory? please guide me on it. also which version is compatible of agensgraph and PostgreSQL

**Comment by Halkrine** [comment]
If you want use hybrid query, it must remain graph database.

So, your data is all relational data, you can't use hybrid query because hybrid query of Agensgraph execute Graph data and Relational data.

So you want use using hybrid query, you convert your relational data to graph data.

**Comment by AnwarsaeedDMU** [comment]
@Halkrine  yes but we can use Agensgraph as extension for PostgreSQL yes? we don't need to convert all data to agensgraph just some or the main table data such as id and name to create node and relationship and keep the remaining attributes data in tables. Some data in graph and some in relational database and we can get result from both at same time using hybrid querying.
